cmd/internal/cli/key_newpair.go

```diff
@@ -31,3 +35,9 @@ var KeyNewPairCmd = &cobra.Command{
 	Long:    docs.KeyNewPairLong,
 	Example: docs.KeyNewPairExample,
 }
+
+// doKeyNewpairCmd is the 'singularity key newpair' command
+func doKeyNewpairCmd(keyServerURI, authToken string) error {
+	_, err := sypgp.GenKeyPair(keyServerURL, authToken)
```

Comment:
Was `keyServerURI` or `keyServerURL` indended? The latter being a global variable in this package.

Reply:
I think the `keyServerURL` is the global variable in this package... Should it be renamed to `keyServerURI`?

Reply:
I think either is acceptable, though `URL` is probably more accurate in this case

Reply:
this should be keyServerURI, the argument to the function. The argument to the function comes from keyServerURL (L#27 above), but for testability's sake this should not use globals.